yarn-project/circuit-types/src/logs/tx_l2_logs.ts

```diff
@@ -1,5 +1,8 @@
+import { sha256 } from '@aztec/foundation/crypto';
 import { BufferReader, prefixBufferWithLength } from '@aztec/foundation/serialize';
 
+import isEqual from 'lodash.isequal';
```

Comment:
@spalladino if I remember correctly Charlie forced you to nuke the lodash dependency and you had a PR removing it (can't find it now). Should we avoid using it?

Reply:
We didn't remove _all_ lodash occurrences, only the ones that were easy to replace with other idioms or our custom versions of them. Properly implementing deep equality is not easy, so we kept that one. PR was https://github.com/AztecProtocol/aztec-packages/pull/3863.

Reply:
Ok, makes sense. Thanks